radicale/auth/ldap.py

```diff
@@ -130,8 +134,8 @@ def __init__(self, configuration: config.Configuration) -> None:
             else:
                 logger.info("auth.ldap_ssl_ca_file     : (not provided)")
         """Extend attributes to to be returned in the user query"""
-        if self._ldap_groups_attr:
-            self._ldap_attributes.append(self._ldap_groups_attr)
+        if self._ldap_groups_attr == "memberOf":
+            self._ldap_attributes.append("memberOf")
```

Comment:
Sorry, this is not acceptable.
In order to fix "perceived" hard-coded values (which in reality are configurable), you now do hard-coding.
This breaks all setups that do not have ActiveDirectory's `memberof` attribute as an indicator of the user side.
Among them: 
* all setups using Novell eDirectoy's `groupMemberShip`
  There may be other directory servers using different schemes, but Novell eDirectory I know for sure.
  OpenLDAP even allows using a freely defined attribute for the user-side of groups (by using a modules)
  * all setup that use a non-DN valued attribute (like .e.g mine)